samples/OpenIdConnectSample/Startup.cs

```diff
@@ -102,7 +98,6 @@ await context.Authentication.SignOutAsync(OpenIdConnectDefaults.AuthenticationSc
                     {
                         RedirectUri = "/signedout"
                     });
-                    await context.Authentication.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
```

Comment:
Reverted, but moved before OIDC, as trying to update the response headers after writing to the response stream (e.g when `AuthenticationMethod = OpenIdConnectRedirectBehavior.FormPost`) is not safe and will likely throw an exception.